doc/pairs.md

```diff
@@ -0,0 +1,35 @@
+# Pairs examples
+
+## Lua Fun
+
+Queries using pairs support the [Lua Fun](https://github.com/luafun/luafun) library. Some examples of working with basic functional functions below.
+
+**Filter example:**
+```lua
+local objects = {}
+for _, obj in fun.filter(function(x) return x.age % 5 == 0 end, crud.pairs('customers', {{'==', 'name', 'Alexey'}}, {use_tomap = true})) do
```

Comment:
I think it's better to rewrite to
```lua
pairs('customers', {{'==', 'name', 'Alexey'}}, {use_tomap = true})):filter(function(x) return x.age % 5 == 0 end)
```

See https://github.com/tarantool/crud/blob/2bb0c29dd8453dba27053143f967ac273e8cc65e/test/integration/pairs_test.lua#L359 for example